ComponentTextKit/TextKit/CKTextKitAttributes.h

```diff
@@ -104,6 +109,7 @@ struct CKTextKitAttributes {
     && maximumNumberOfLines == other.maximumNumberOfLines
     && shadowOpacity == other.shadowOpacity
     && shadowRadius == other.shadowRadius
+    && layoutManagerFactory == other.layoutManagerFactory
```

Comment:
Seems like we should probably include the layout manager factory in the hashing function? https://github.com/facebook/componentkit/blob/master/ComponentTextKit/TextKit/CKTextKitAttributes.mm#L20